main.go

```diff
@@ -111,16 +112,20 @@ func main() {
 		os.Exit(1)
 	}
 
+	renewDeadline := time.Duration(leaderRenewSeconds) * time.Second
+	leaseDuration := time.Duration(int(1.2*float64(leaderRenewSeconds))) * time.Second
+
 	// Load controller manager configuration and create manager options from
 	// it.
 	ctrlConfig := configstorageoscomv1.OperatorConfig{}
 	options := ctrl.Options{
-		Scheme:                  scheme,
-		LeaderElectionID:        "storageos-operator-leader",
-		LeaderElectionNamespace: currentNS,
-		RenewDeadline:           &renewDeadline,
-		LeaseDuration:           &leaseDuration,
-		RetryPeriod:             &leaderRetryDuration,
+		Scheme:                     scheme,
+		LeaderElectionID:           "storageos-operator-leader",
+		LeaderElectionNamespace:    currentNS,
+		LeaderElectionResourceLock: "leases",
```

Comment:
nit: I believe you can use the constant k8s.io/client-go/tools/leaderelection/resourcelock. LeasesResourceLock